MediaGallerySynchronization/Model/FileEncryption/Sha1File.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGallerySynchronization\Model\FileEncryption;
+
+use Magento\MediaGallerySynchronizationApi\Model\FileEncryptionInterface;
+
+class Sha1File implements FileEncryptionInterface
```

Comment:
I don't think this additional class and interface are needed. It's enough to have only GetContentHash and GetContentHashInterface.